sync-quickstart-android/src/main/java/com/twilio/sync/quickstart/TicTacActivity.java

```diff
@@ -98,6 +109,26 @@ public void onClick(View v) {
         authenticateAndStartSync();
     }
 
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        getMenuInflater().inflate(R.menu.tictak, menu);
+        return super.onCreateOptionsMenu(menu);
+    }
+
+    @Override
+    public boolean onOptionsItemSelected(MenuItem item) {
+        switch (item.getItemId()) {
+            case R.id.action_crash_in_sync_client:
+                SyncClientUtils.simulateCrash(syncClient, SYNC_CLIENT_CPP);
```

Comment:
I'm not sure if it should end up in public demo application.

Reply:
It should. It demonstrates that crash symbolication works. BTW same changes already merged into public chat demo